src/distributions/uniform.rs

```diff
@@ -66,9 +66,9 @@ impl<X: SampleUniform> Uniform<X> {
     }
 
     /// Create a new `Uniform` instance which samples uniformly from the closed
-    /// range `[low, high]` (inclusive). Panics if `low >= high`.
+    /// range `[low, high]` (inclusive). Panics if `low > high`.
     pub fn new_inclusive(low: X, high: X) -> Uniform<X> {
-        assert!(low < high, "Uniform::new called with `low >= high`");
+        assert!(low <= high, "Uniform::new_inclusive called with `low > high`");
```

Comment:
Good idea to solve #403. I looked over the code and don't see any issues with `low == high`.

Reply:
Yes. `low == high` is nonsensical, but not impossible.

Reply:
It is the trivial case, but but it is well-defined and supporting it might simplify more general code.